.circleci/deploy-qa.sh

```diff
@@ -13,7 +13,7 @@ pipenv install
 # install playbook role requirements
 pipenv run ansible-galaxy install -r requirements.yml
 # setup vault password retrieval from travis envvar
-cp .circleci/.vault ~/.vault
+$ANSIBLE_VAULT_PASSWORD > ~/.vault
```

Comment:
This should echo the password
```suggestion
echo $ANSIBLE_VAULT_PASSWORD > ~/.vault
```

But be careful, using environment variables out in the open like this can leave potentially leak the password.  I'm guessing that's why whoever set this up first used a file instead of an environment variable.